### DISCUSSION TITLE
Grid Search

### DISCUSSION THREAD

**Discussion by ali-imdad-khan** [yandex-research/rtdl-revisiting-models#32] [Q&A] [answered]
Hello :) Can you please suggest how i can implement grid search for hyperparamter optimization  according to the tutorial ?

**Comment by Yura52** [comment, marked as answer]
Hey! You can see how we perform grid search for NODE, e.g. for the California Housing dataset here: [link](https://github.com/Yura52/tabular-dl-revisiting-models/tree/main/output/california_housing/node/tuning/0).

So, you just have to create all the configs from your grid, run with all of them one by one, and select the best one **based on the validation score**.
